### DISCUSSION TITLE
confused between Kafkaconnector and Kafkaconnect

### DISCUSSION THREAD

**Discussion by hari819** [strimzi/strimzi-kafka-operator#6961] [Q&A]
Please can someone guide me on the difference between Kind: KafkaConnector and Kind: KafkaConnect ,

**Comment by hari819** [comment]
basically i am trying to build and run an ElasticSink plugin on top of KafkaConnect base image from Strimzi , to achieve functionality which can read from a topic and push to ES ,  i am using this ES Jar file , https://packages.confluent.io/maven/io/confluent/kafka-connect-elasticsearch/11.1.10/

**Reply by scholzj** [reply]
`KafkaConnect` represents the Kafka Connect server. `KafkaConnector` represents the Connector instance running on top of the Kafka Connect cluster. So with `KafkaConnect` you deploy the Kafka Connect cluster with the connector plugins you plan to use. And then with `KafkaConnector` you can create the connector instances whicih will be scheduled on the Connect cluster.

**Reply by hari819** [reply]
thankyou @scholzj for the clarification , i will verify the examples shared in the repo
